### DISCUSSION TITLE
v2.0.24

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#274] [Announcements]
## Changes

- Query should do nothing if empty @DTTerastar (#273)
- Improve room assistant detection @DTTerastar (#268)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.24'>v2.0.24</a>.</em>

**Comment by dodgeboy** [comment]
<img width="367" alt="image" src="https://user-images.githubusercontent.com/18379759/153925429-f76a3e4a-d44f-4189-98a6-5d94aee8acbe.png">
I get this error when trying to install from the installation page, ever since 2.0.24 was released.  I'm hoping to be able to get all my sensors back online after 2.0.23 broke them all.

**Reply by DTTerastar** [reply]
GIthub actions takes 20 minutes to build and attach all the images.  I just tried and they are now up.

**Reply by dodgeboy** [reply]
Thanks!  I appreciate the quick turnaround on this release!  I'm still getting the same error, but I will give it a bit more time.
